Configurations/unix-Makefile.tmpl

```diff
@@ -654,9 +654,11 @@ errors:
 	( cd $(SRCDIR); $(PERL) util/ck_errf.pl -strict */*.c */*/*.c )
 	( cd $(SRCDIR); $(PERL) util/mkerr.pl -internal )
 	( cd $(SRCDIR)/engines; \
-	  for e in *.ec; do \
-	      $(PERL) ../util/mkerr.pl -conf $$e -rebuild *.c; \
-	  done )
+          $(PERL) ../util/mkerr.pl -conf e_capi.ec e_capi.c ; \
+          $(PERL) ../util/mkerr.pl -conf e_dasync.ec e_dasync.c ; \
+          $(PERL) ../util/mkerr.pl -conf e_ossltest.ec e_ossltest.c ; \
+          cd afalg; \
+          $(PERL) ../../util/mkerr.pl -conf e_afalg.ec e_afalg.c )
```

Comment:
That would be silly, we basically already have that information.  The `.ec` files go together with end product files, right?  Like libraries, DSOs (possibly programs, but we don't use them like that), right?  So what we would do is to tell the the build file builder that `crypto/err/openssl.ec` deals with every source file used to build `libcrypto` and `libssl`, etc etc.  We'd end up with something like this (not a finished thought, but where I'm currently heading with this):

in the top `build.info`:
```
EC[libcrypto]=crypto/err/openssl.ec
EC[libssl]=crypto/err/openssl.ec
```

In `engines/build.info`:
```
EC[capi]=e_capi.ec
EC[dasync]=e_dasync.ec
EC[ossltest]=e_ossltest.ec
EC[padlock]=e_padlock.ec
```

In `engines/afalg/build.info`:
```
EC[afalg]=e_afalg.ec
```

The `update` target can pick up from that...